package.json

```diff
@@ -42,6 +42,7 @@
     "prettier:all": "yarn babel-node ./scripts/prettier.js write",
     "size": "size-limit",
     "size:overhead:why": "size-limit --why ./test/size/overhead.js",
+    "size:snapshot": "node scripts/createSizeSnapshot",
     "size:why": "size-limit --why packages/material-ui/build/index.js",
```

Comment:
The best command ever! We might be able to kill 1KB by removing the usage of `dom-helpers/styles` :) cc @joshwooding 

![capture d ecran 2019-02-21 a 10 55 01](https://user-images.githubusercontent.com/3165635/53160152-a1c02900-35c7-11e9-9969-26cc7011ebf3.png)